lib/component.js

```diff
@@ -85,7 +86,9 @@ export default class Component extends WebComponent {
   }
 
   update(stateUpdate={}) {
-    if (this.isPanelRoot) {
+    if (typeof this.app === 'undefined') {
```

Comment:
let's just use an explicit prop that gets set during the `attachedCallback`, e.g., `if (this.initialized) {`